P0009/P0009.md

```diff
@@ -2957,30 +2964,19 @@ constexpr size_type size() const;
 [8]{.pnum} *Returns:* `extents().`_`fwd-prod-of-extents`_`(rank())`.
 
 ```c++
-constexpr void swap(mdspan& y) noexcept;
+friend constexpr void swap(
+  mdspan<ElementType, Extents, LayoutPolicy, AccessorPolicy>& x,
```

Comment:
I don't think you need the template parameters here (unless LWG asked for them).